page_objects/page_ovirt_hostedengine.py

```diff
@@ -259,3 +262,211 @@ def check_additional_host_socre(self, ip, passwd):
                 break
             time.sleep(10)
             i += 1
+
+    def default_vm_engine_stage_config(self):
+        # VM STAGE
+        self.click(self.HE_START)
+        self.input_text(self.VM_FQDN, self.config_dict['he_vm_fqdn'])
+        self.input_text(self.MAC_ADDRESS, self.config_dict['he_vm_mac'])
+        self.input_text(self.ROOT_PASS, self.config_dict['he_vm_pass'])
+        self.click(self.NEXT_BUTTON)
+
+        # ENGINE STAGE
+        self.input_text(self.ADMIN_PASS, self.config_dict['admin_pass'])
+        self.click(self.NEXT_BUTTON)
+
+        # PREPARE VM
+        self.click(self.PREPARE_VM_BUTTON)
+        self.click(self.NEXT_BUTTON, 1500)
+
+    def node_zero_default_deploy_process(self):
+        def prepare_env():
+            self.move_failed_setup_log()
```

Comment:
I suggest to encapsulate only one prepare_env method for all processes, this method should deal with cleaning of different storages

Reply:
solved